hx_lti_initializer/templates/hx_lti_initializer/instructor_index.html

```diff
@@ -0,0 +1,167 @@
+                {{ course.course_name }}
+            </div>
+            {% if 'Administrator' in roles or 'Instructor' in roles %}
+            <a class="settingslink" href="{{ course.get_admin_url }}" target="_blank">Change Course Settings</a>
+            {% endif %}
+            <div id="copypaste-{{course.id}}"></div>
+            <table id="assignments" class="table table-striped">
+                <tbody>
+                {% for assignment in course.assignment_set.all %}
+                <tr data-toggle="collapse" data-target="#target-objects-{{ assignment.id }}" class="clickable">
+                    <td>
+                        {% if 'Administrator' in roles or 'Instructor' in roles %}
+                        <a class="settingslink leftfloat" href="{{ assignment.get_admin_url }}" target="_blank">Edit</a>
+                        {% endif %}
+                        {{ assignment.assignment_name }}
+                        <span class="expandicon"></span>
+                    </td>
+                </tr>
+                <tr>
+                    <td>
+                        <table class="table table-striped collapse" id="target-objects-{{ assignment.id }}">
+                            <thead>
+                            <tr>
+                                <th>
+                                    Title
+                                </th>
+                                <th>
+                                    Author
+                                </th>
+                                <th>
+                                    Date Created
+                                </th>
+                                <th></th>
+                            </tr>
+                            </thead>
+                            <tbody>
+                            {% for file in assignment.assignment_objects.all %}
+                            <tr>
+                                <td>
+                                    {% if 'Administrator' in roles or 'Instructor' in roles %}
+                                    <a class="settingslink leftfloat" href="{{ file.get_admin_url }}" target="_blank">Edit</a>
+                                    {% endif %}
+                                    {{ file.target_title }}
+                                    <!--<a href="#" onclick="printHelp('{{ course.id }}', '{{ assignment.assignment_id }}', '{{ file.id }}'); return false;">{{ file.target_title }}</a>-->
+                                </td>
+                                <td>
+                                    {{ file.target_author }}
+                                </td>
+                                <td title="{{ file.target_created }}">
+                                    {{ file.target_created | timesince }} ago
+                                </td>
+                                <td>
+                                    <form action="annotation_view" id="widget_edit_form" method="post">
```

Comment:
Any particular reason why this couldn't just be a standard link? It would be better to make this a standard link unless there's a reason why it needs to POST to the view.